### ISSUE TITLE
Getting Errors when running App.

### ISSUE THREAD

**Issue by Mike-LeGall** [checkandmate1/AirplaneFetcher#1]
App fails to execute. Not sure if it's something on my end. 
I created a folder named resources and placed the openscope-airlines.json file in it. The json was copied from vice.

Here are the errors that come up. 

C:\Users\rivet>"D:\airplanefetcher\flightplanfiller.exe" -airport KEWR -amount 50
Fetch Callsigns 100 % [==============================================================================] Finished!
Fetch Departures   0 % [------------------------------------------------------------------------------] 00:03
Fetch Arrivals   0 % [------------------------------------------------------------------------------] 0s
panic: open resources/openscope-airlines.json: The system cannot find the path specified.

goroutine 162 [running]:
main.LoadResource(...)
        D:/a/AirplaneFetcher/AirplaneFetcher/parseOpenscope.go:37
main.parseAirlines()
        D:/a/AirplaneFetcher/AirplaneFetcher/parseOpenscope.go:52 +0x577
main.flightAwareNonsenseDepartures({0xc0002b6000, 0xe0c, 0x439d60?}, 0x32, 0xc000024540)
        D:/a/AirplaneFetcher/AirplaneFetcher/main.go:118 +0x56a
created by main.getDepartureCallsigns2 in goroutine 1
        D:/a/AirplaneFetcher/AirplaneFetcher/main.go:324 +0x1c3b

Thanks for the help

**Comment by mumuwawa** [comment]
It looks like the application is unable to find the openscope-airlines.json file in the resources directory. Here are a few things you can check:

Make sure that the resources folder is located in the same directory as your executable (flightplanfiller.exe) and that the filename is exactly openscope-airlines.json. Verify there are no typos or extra spaces.

Ensure you are running the command from the correct directory where the resources folder is located. You might also try using an absolute path for the resources folder in your command to ensure it’s pointing to the correct location. For example:


"D:\airplanefetcher\flightplanfiller.exe" -airport KEWR -amount 50 -resources "D:\airplanefetcher\resources\openscope-airlines.json"

Check the file and directory permissions to make sure that the application has the necessary access to read the file. Additionally, ensure that the openscope-airlines.json file is not corrupted or empty by opening it in a text editor.

If you have access to modify the code, adding debugging statements to print the current working directory and the path being accessed could help identify any issues with the file path being used by the application.

**Comment by Mike-LeGall** [comment]
I rebuilt everything to the root Directory. The program doesn't recognize the -resources tag. 

D:\>"D:\airplanefetcher\flightplanfiller.exe" -airport KEWR -amount 100 -resources "D:\airplanefetcher\resources\openscope-airlines.json"
flag provided but not defined: -resources
Usage of D:\airplanefetcher\flightplanfiller.exe:
  -airport string
        airport to fetch
  -amount string
        amount of aircraft

Tried to run it from Command prompt and power shell with and without admin rights. Unfortunately I don't have access/skill to add a debug statement,
Thanks again

**Comment by Mike-LeGall** [comment]
I got it to work. As you said 'Ensure you are running the command from the correct directory where the resources folder is located'. I was actually running it from the root directory and not the D:\airplanefetcher\ directory that I had created to put the program in. Would it be possible for the program to recognize what folder it's in and then look for the resources folder? just a thought. Awesome program, what a time saver. Thanks again..
